ios/Classes/FlutterWebviewPlugin.m

```diff
@@ -32,27 +49,97 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     }
 }
 
-- (void)showWebView:(FlutterMethodCall*)call {
+- (void)launch:(FlutterMethodCall*)call {
     NSString *url = call.arguments[@"url"];
-    NSNumber *withJavascript = call.arguments[@"withJavascript"];
+    
+    NSURLRequest *request = [NSURLRequest requestWithURL:[NSURL URLWithString:url]];
+    [self.webview loadRequest:request];
+}
+
+- (void)initWebView:(FlutterMethodCall*)call {
+    // NSNumber *withJavascript = call.arguments[@"withJavascript"];
     NSNumber *clearCache = call.arguments[@"clearCache"];
     NSNumber *clearCookies = call.arguments[@"clearCookies"];
-    NSNumber *fullScreen = call.arguments[@"fullScreen"];
+    NSNumber *hidden = call.arguments[@"hidden"];
```

Comment:
could you document this ? removing `fullscreen` is a breaking change and I need to know if I can do the same thing for Android

Reply:
OK. I will add some document.

fullScreen is default.